lib/repl.js

```diff
@@ -1058,7 +1058,7 @@ REPLServer.prototype.defineCommand = function(keyword, cmd) {
     cmd = {action: cmd};
   } else if (typeof cmd.action !== 'function') {
     throw new errors.TypeError('ERR_INVALID_ARG_TYPE',
-      'action', 'function', cmd.action);
+                               'action', 'function', cmd.action);
```

Comment:
ah, doesn't make much of a difference to me, so up to you. I was mainly just curious. Maybe that was just for cpp code.